src/components/lobby/Lobby.js

```diff
@@ -39,6 +40,54 @@ export class Lobby extends React.Component {
     componentDidMount() {
     }
 
+    async startGame()
```

Comment:
Do we check server-side if it's the lobby host? Because the others can't start the game.

Reply:
Today only they merged the server side code for this. thanks for reminding me. I will check whether it is the lobby host